.travis.yml

```diff
@@ -1,9 +1,25 @@
 language: python
 python:
-  - "2.7"
+  - 2.7
+  - 3.5
+  - 3.6
+matrix:
+  - python: 3.7
+    dist: xenial
 install:
   - easy_install vsc-install
+  # installing coloredlogs and future is only needed for the import tests
+  # this can be removed if only "python setup.py test" is used (since that auto-installs required dependencies)
+  - if [ "$(echo $TRAVIS_PYTHON_VERSION | cut -f1 -d.)" == '3' ]; then pip install coloredlogs future; fi
 script:
   # 'hpcugent' is a mandatory remote for setup.py to work (because of vsc-install)
   - git remote add hpcugent https://github.com/hpcugent/vsc-base.git
-  - python setup.py test
+  # only run part of the tests with Python 3 for now (until whole code base is Py3 compatible)
+  - if [ "$(echo $TRAVIS_PYTHON_VERSION | cut -f1 -d.)" == '3' ]; then
+        set -e;
+        export PYTHONPATH="$PWD/lib:$PYTHONPATH";
+        python -c "import vsc.utils.fancylogger";
+        python -c "import test.fancylogger";
```

Comment:
this does not actually test anything, so why bother?

Reply:
It saves some of the problems, makes the PR short and easy to review, etc., the tests pass as is, etc.

Baby steps.

Reply:
To clarify:

* this checks whether `vsc/utils/fancylogger.py` no longer has `SyntaxError`s; same for `test/fancylogger.py`
* can't run only the `fancylogger` subsuite yet, more changes are needed for that due to `SyntaxError` in `test/missing.py`

If you prefer, I can fix the import `test/missing.py` too, and see if that's enough to run the `fancylogger` tests with Python 3 (but it's going to blow up the PR, why not merge it first since it's easy to review right now).